### PR TITLE
Falsh Will be bought last

### PR DESCRIPTION
**Pull request by sebbi08**
Falsh Will be bought last becaus if u picked up one last round and buy it befor other nades u will get 2 flashes insted of HE or Somoke

### CODE REVIEW

File: js/buynds.js [hunk 1]
```diff
@@ -48,14 +48,30 @@
...
                 bindOptions.grenades.forEach(function (grenade) {
                     var grenadeArray = grenade.split(',');
                     grenadeArray.forEach(function (nade) {
-                        bindString += 'buy ' + nade + '; ';
+                        granadeStringArray[count] = 'buy ' + nade + '; ';
+                        count++;
                     });
                 });
             }
+            var granadeString ="";
```
Comment (mike-u) [inline js/buynds.js]: Any reason for not calling it grenadeString?

**Comment by mike-u** [comment]
I guess the travis build needs to know to expect the flash to be last?

File: js/buynds.js (hunk 1 above)
Comment (sebbi08) [inline js/buynds.js]: y i am dyslexic so just a typo thx for noticing it is fixed

**Comment by mike-u** [comment]
Now travis is confused by the name change. But before that change one error it threw was

```
Expected 'bind "kp_5" "buy incgrenade; buy molotov;";' to equal 'bind "kp_5" "buy molotov; buy incgrenade;";'.
    @/home/travis/build/jessewebb/csgo-buynds/tests/specs/buynds.spec.js:85:17
```

Why would this change in order happen? There's no flash being bought.

File: js/buynds.js [hunk 2]
```diff
@@ -48,14 +48,30 @@
...
+            var granadeStringArray = [];
+            var count = 0;
+
+
             if (bindOptions.grenades) {
                 bindOptions.grenades.forEach(function (grenade) {
                     var grenadeArray = grenade.split(',');
                     grenadeArray.forEach(function (nade) {
-                        bindString += 'buy ' + nade + '; ';
+                        granadeStringArray[count] = 'buy ' + nade + '; ';
```
Comment (mike-u) [inline js/buynds.js:59]: This typo might be the problem, should be grenadeStringArray

**Comment by jessewebb** [comment]
The code was already written to honor the order of grenades that the user clicks on when configuring the bind. For instance:
- If the user first clicks on "Flashbang" and then "Decoy" and then the "x 2" for a second flashbang, the generated bind will be: `buy flashbang; buy decoy; buy flashbang;`
- If the user first clicks on "Smoke" and then "Flashbang" and then the "x 2" ad then "Decoy", the generated bind will be: `buy smokegrenade; buy flashbang; buy flashbang; buy decoy;`

The generator keeps track of which order the user clicks on the check-boxes and generates the order of the bind accordingly.

I do not wish to change this behavior. Closing this PR.

**Pull request closed without merging**